api-java-mixin.raml

```diff
@@ -260,6 +450,82 @@ types:
+    (java-mixin): |
+      public static ProductSetImageLabelAction ofUnset(final Long variantId, final String imageUrl, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetImageLabelActionBuilder.of().variantId(variantId).imageUrl(imageUrl).staged(staged).build();
+      }
+
+      public static ProductSetImageLabelAction ofUnsetStaged(final Long variantId, final String imageUrl) {
+          return ProductSetImageLabelActionBuilder.of().variantId(variantId).imageUrl(imageUrl).build();
+      }
+  ProductSetTaxCategoryAction:
+    (java-mixin): |
+      public static ProductSetTaxCategoryAction ofUnset() {
+          return ProductSetTaxCategoryAction.of();
+      }
+  ProductSetCustomerAction:
+    (java-mixin): |
+      public static ProductSetCustomerAction ofUnset() {
+          return ProductSetCustomerAction.of();
+      }
+  ProductSetProductPriceCustomFieldAction:
+    (java-mixin): |
+      public static ProductSetProductPriceCustomFieldAction ofUnset(final String name, final String priceId) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).build();
+      }
+
+      public static ProductSetProductPriceCustomFieldAction ofUnsetStaged(final String name, final String priceId,
+              @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).staged(staged).build();
+      }
+  ProductSetAttributeAction:
+    (java-mixin): |
+      public static ProductSetAttributeAction ofVariantId(final Long variantId, final String name, @javax.annotation.Nullable final com.fasterxml.jackson.databind.JsonNode value, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).staged(staged).build();
+      }
+
+      public static ProductSetAttributeAction ofSku(final String sku, final String name, @javax.annotation.Nullable final com.fasterxml.jackson.databind.JsonNode value, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().sku(sku).name(name).value(value).staged(staged).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttribute(final Long variantId, final String name) {
+          return ofUnsetAttribute(variantId, name, null);
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttribute(final Long variantId, final String name, @javax.annotation.Nullable final Boolean staged) {
+          return ofVariantId(variantId, name, null, staged);
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttributeForVariantId(final Long variantId, final String name) {
+          return ofUnsetAttributeForVariantId(variantId, name, null);
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttributeForVariantId(final Long variantId, final String name, @javax.annotation.Nullable final Boolean staged) {
```

Comment:
seems to be duplicate of `ofUnsetAttribute(final Long variantId, final String name, @javax.annotation.Nullable final Boolean staged)`

Reply:
I'm also removing it